StellarApp/Protocol/OrbitRotationFiles.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  OrbitRotationFiles.swift
+//  StellarApp
+//
+//  Created by TingxinLi on 5/1/19.
+//  Copyright © 2019 TingxinLi. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ARKit
+
+struct RotationVector {
+    let x: CGFloat, y: CGFloat, z: CGFloat
+}
+
+protocol Rotatable {
+    func rotate()
+    func stopRotating()
+}
+
+class RotatingNode: SCNNode, Rotatable {
```

Comment:
if we're not using this stuff you can go ahead and delete it. If we need it later on it'll be in the git history

Reply:
ok

Reply:
I want to keep it for now till we talk about tomorrow.